src/app/administrator/admin-book/admin-book.component.html

```diff
@@ -1,3 +1,122 @@
-<p>
-  admin-book works!
-</p>
+<button type="button" (click)="openWriteModal('Create', writeBookModal)">
+  Create
+</button>
+<table class="table">
```

Comment:
This will all look good when you add the appropriate bootstrap classes. Shouldn't take too long.

Reply:
We should all collaborate on what classes to use for the buttons and tables. Currently I just have the "table" class for my tables and I think it looks pretty adequate. For buttons I'm thinking something like black for delete, dark gray for update, and light gray for create would look good.